modules/actions/add_member.js

```diff
@@ -1,6 +1,7 @@
+import { utilArrayGroupBy, utilObjectOmit } from '@id-sdk/util';
```

Comment:
Just a comment- since these are in a 'util' namespace, is it redundant to keep the 'util' method name prefix?

Reply:
Yeah that is a good point - This kind of gets into the API design of the iD-sdk, so that's the place where we'd discuss what things are named.

A long time ago, the iD code just concated itself into a global `iD` object, and we grouped things into namespaces like `iD.action.move`, `iD.mode.move`, `iD.operation.move`. 

When we started organizing things into ES6 modules and using rollup to bundle all the code, this because a problem, because all modules get hoisted into a common scope, so I needed to disambiguate all the names of the things to be like `iD.actionMove`, `iD.modeMove`, `iD.operationMove`.  (D3.js went through a similar transition when they adopted rollup and ES6 modules, and I sort of followed their lead).  https://github.com/openstreetmap/iD/pull/3479 this was the main breaking change that took us from iD v1.x to iD v2.x.